### DISCUSSION TITLE
Nominating @TisVictress for Contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#143] [Python Team]
I'd like to nominate @TisVictress for the Contributor role on the python team. She has made a lot of useful contributions to python buildpacks recently and has demonstrated the intent to continue doing so.

@paketo-buildpacks/python-maintainers

**Comment by ryanmoran** [comment]
I think we have 2/3 of @paketo-buildpacks/python-maintainers. Is that right?

**Comment by joshuatcasey** [comment]
We now have 3/3 :)

**Comment by ryanmoran** [comment]
Congrats @TisVictress on becoming a Python Contributor! 🎉
